### DISCUSSION TITLE
filter

### DISCUSSION THREAD

**Discussion by shellduck8** [sherlock-project/sherlock#1384] [Q&A]
Is there a way to filter the program so it only searches in a specific website?
![hacker](https://user-images.githubusercontent.com/81569204/175101836-bae932e7-231c-48eb-93ec-1ca0013addb0.gif)

**Comment by karimkohel** [comment]
It's documented in the [README usage section](https://github.com/sherlock-project/sherlock#usage) that you can add the --site flag for limiting the search to the given sites
